internal/pkg/service/buffer/store/model/file.go

```diff
@@ -31,10 +30,8 @@ type File struct {
 	LastError       string                         `json:"lastError,omitempty"`
 	RetryAttempt    int                            `json:"retryAttempt,omitempty"`
 	RetryAfter      *utctime.UTCTime               `json:"retryAfter,omitempty"`
-	// Statistics are set by the "file close" operation, the value is nil, if there is no record.
-	Statistics *Stats              `json:"statistics,omitempty"`
-	IsEmpty    bool                `json:"isEmpty,omitempty"`
-	StorageJob *keboola.StorageJob `json:"storageJob,omitempty"`
+	IsEmpty         bool                           `json:"isEmpty,omitempty"`
+	StorageJob      *keboola.StorageJob            `json:"storageJob,omitempty"`
```

Comment:
Z `File` a z v dalsom commite zo `Slice` som odstranil `Statistics` field.

**Toto bol hlavny problem v produkcii.**

Ako pisem v dokumentacii, `L1 cache` (predtym iba `cache`) ma v memory kopiu vsetkych statistik, ziskanu cez etcd watch API. 

No povodne to nebol jeden `stats/` prefix, ale 2.
- V `stats/` sa ukladali statistiky per slice a API node, pre slice do ktorych sa aktivne zapisovalo.
- No a ked sa zapis ukoncil, tak sa statistiky presunuli do `Slice` a po importe sa ulozila kopia aj do `File`.
- Mysleli sme si, ze je to elegantne riesenie.
- Problem ale bol s tou `cache`, mali sme tak 2 etcd watch streams, ktore sa ale nedaju nijako synchronizovat.
- Problematicky je hlavne presun zo `/stats` do `Slice`, ... v jednom streame pride `Delete` event a v druhom `Put`.
- Ale v nahodnom poradi, ... bola snaha to nejak vyriesit, ale neporadilo sa to uplne, ... staci nejaky mensi sietovy vypadok a cache nebude vracat atomicke vysledky, co napr. zablokuje klientovi import na `quota`, lebo sa napr. nespracoval nejaky `delete` event a cache si mysli, ze je tam toho viac ako realne.

https://github.com/keboola/keboola-as-code/blob/be9cdf7c28fc79c8e86c73501b9b58571ab05c5b/internal/pkg/service/buffer/statistics/cache/cache.go#L129-L199